plinth/modules/mediawiki/__init__.py

```diff
@@ -40,12 +40,12 @@
       'edited website. You can use MediaWiki to host a wiki-like website, '
       'take notes or collaborate with friends on projects.'),
     _('This MediaWiki instance comes with a randomly generated administrator '
-      'password. You can set a new password in the Configuration section and '
-      'login using the "admin" account. You can then create more user '
+      'password. You can set a new password in the configuration section and '
```

Comment:
`Configuration` is the name of the section and it is used as a proper noun here. Probably removing the `the` before it is more appropriate.

How about this?
`You can set a new password in the section, Configuration, and`

Reply:
It would need to be quoted to be used with the big c. Possibly this is the convention used elsewhere, I have since forgotten whether i checked.

Reply:
Agreed. Can be changed to one of the following.

`You can set a new password in the section Configuration, and`

or

`You can set a new password in the "Configuration" section and`